crates/solidity/outputs/cargo/slang_solidity/src/generated/bindings/generated/built_ins.rs

```diff
@@ -0,0 +1,58 @@
+// This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+use semver::Version;
+
+#[allow(unused_variables)]
+pub fn get_contents(version: &Version) -> &'static str {
+    if *version < Version::new(0, 5, 0) {
+        r####"contract $$ {
```

Comment:
Instead of using `built_ins.rs.jinja2`, I suggest generating separate files for each version:

- `slang_solidity/src/generated/bindings/builtins/0.4.11.sol`
- `slang_solidity/src/generated/bindings/builtins/0.5.0.sol`
- etc...

And include it here via `include_str()!`:

```rs
#[allow(unused_variables)]
pub fn get_contents(version: &Version) -> &'static str {
    if *version < Version::new(0, 5, 0) {
        include_str("./builtins/0.4.11")
    } else {
      // etc...
    }
}
```

This allows diffing the versions and comparing changes during code review.